### PR TITLE
Option for not setting up evil keys

### PR DESCRIPTION
**Pull request by angerborn**
First I want to say, thanks, fantastic package!

This MR adds an optional argument to `eyebrowse-setup-opinionated-keys` to avoid setting up the key bindings for evil. Using evil doesn't mean that you'd want these binds necessarily.

As it's an optional argument and set to ignore setting evil bindings if set, it doesn't change the api of this method and should work exactly the same for everyone, unless they're specifying this argument.

What do you think, is it reasonable or do you suggest some other solution (that may or may not involve changing eyebrowse.el).

### CODE REVIEW

**Comment by wasamasa** [comment]
Thanks for the PR! The mechanism looks good to me. One thing I'd change though, the docstring is a bit confusing and doesn't quite follow how it works. I'd rephrase it to something like "If IGNORE-EVIL isn't set and Evil is detected, extra key bindings will be set up with `eyebrowse-setup-evil-keys' as well."

**Comment by angerborn** [comment]
Thanks for taking the time to look and answer. I agree it was a bit confusing. I've pushed an update with the doc string you suggested instead.
